runtime/android/core/src/org/xwalk/core/XWalkActivity.java

```diff
@@ -177,12 +178,21 @@ private void initXWalkLibrary() {
         int status = XWalkLibraryLoader.initXWalkLibrary(this);
         if (status == XWalkLibraryInterface.STATUS_MATCH) {
             if (mActiveDialog != null) dismissDialog();
+            if (mDecoratedBackground) {
+                getWindow().setBackgroundDrawable(null);
+                mDecoratedBackground = false;
+            }
             XWalkLibraryLoader.startInitialization(mLibraryListener);
             return;
         }
 
         if (mActiveDialog != null) return;
 
+        if (getWindow().getDecorView().getBackground() == null) {
```

Comment:
I think it would be good to add a short comment explaining why this is being done.